scripts/build-release.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e  # Exit immediately if a command exits with a non-zero status.
+
+# Variables
+ZIP_FILE="tumblr-importer.zip"
+
+# Step 1: Install dependencies
+echo "Installing dependencies..."
+composer install --no-dev --optimize-autoloader
+
+# Step 2: Create the zip file
+echo "Creating zip file $ZIP_FILE..."
+zip -r $ZIP_FILE . -x ".*" -x "scripts/*" -x "*.git*" -x "composer.*" -x "phpcs.xml"
```

Comment:
I don't know if it's worth it since we don't need any other scripts from it, but maybe using `wp-scripts plugin-zip` would make sense instead of our own script:

https://developer.wordpress.org/block-editor/reference-guides/packages/packages-scripts/#plugin-zip

Reply:
I've removed this script for good